README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
+
+Install Go Task
+
```

Comment:
Ensure that the installation commands are error-handled. For example, check if 'brew install go-task' succeeded before proceeding to install dependencies. This avoids potential issues if the first command fails and 'task install' is attempted subsequently.